packages/cli/src/commands/ci/migrate-manifest.ts

```diff
@@ -0,0 +1,81 @@
+import {Command} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import color from '@heroku-cli/color'
+
+const fs = require('fs')
+const BB = require('bluebird')
+const writeFile = BB.promisify(fs.writeFile)
+const unlinkFile = BB.promisify(fs.unlink)
+
+export default class CiMigrateManifest extends Command {
+  static description = 'app-ci.json is deprecated. Run this command to migrate to app.json with an environments key.'
+  static topic = 'ci'
+  static examples = [
+    '$ heroku ci:migrate-manifest',
+  ]
+
+  async run() {
+    const appJSONPath = `${process.cwd()}/app.json`
+    const appCiJSONPath = `${process.cwd()}/app-ci.json`
+    let action: string
+
+    function showWarning() {
+      ux.log(color.green('Please check the contents of your app.json before committing to your repo.'))
+    }
+
+    async function updateAppJson() {
+      // Updating / Creating
+      ux.action.start(`${action.charAt(0).toUpperCase() + action.slice(1)} app.json file`)
+      writeFile(appJSONPath, `${JSON.stringify(appJSON, null, '  ')}\n`)
```

Comment:
I think we need an `await` here.
```suggestion
      await writeFile(appJSONPath, `${JSON.stringify(appJSON, null, '  ')}\n`)
```